uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -129,6 +145,31 @@ define([
       });
     });
 
+    describe('$scope.role', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      it('defaults to as admin', function () {
+        expect($scope.role).toBe('admin');
+      });
+    });
+
+    describe('isRole()', function () {
+      beforeEach(function () {
+        initController();
```

Comment:
Impleented following code 
```js
// ...
function initController(scopeValues) {
  $controller('ModalDocumentCtrl', {
    $scope: _.assign($scope, scopeValues),
  }
}
```

But, here, `$scope.role` even passed as `initController({ role: 'staff' });`, it will get overridden by the value of `role`. As the  `$scope.role` is dependent on `role`. So, in this case, we need to do `$scope.role = 'staff';` after controller is initialized.
```js
describe('isRole()', function () {
      beforeEach(function () {
        initController();
        $scope.role = 'staff';
      });

      it('checks if the given role is the current one', function () {
        expect($scope.isRole('staff')).toBe(true);
        expect($scope.isRole('admin')).toBe(false);
      });
    });
```